app/Http/Controllers/Secretariat/UserController.php

```diff
@@ -431,14 +432,15 @@ public function showTenantUpdate()
     }
 
     /**
-     * Updates a tenant to an applicant
+     * Updates a tenant (or a user with no status) to an applicant
      */
     public function tenantToApplicant()
     {
-        if (!user()->isTenant() || user()->isCollegist(alumni: false)) {
-            return abort(403);
+        if (user()->isCollegist(alumni: false)) {
+            return abort(403, 'You are already a collegist.');
         }
         $user = user();
+        $user->update(['verified' => false]);
```

Comment:
applicants might also be verified (see reapplication for resident status). Are you sure you need this?